docs/src/man/getting_started.md

```diff
@@ -229,6 +229,27 @@ Note that constructing a `DataFrame` row by row is significantly less performant
 constructing it all at once, or column by column. For many use-cases this will not matter,
 but for very large `DataFrame`s  this may be a consideration.
 
+If we wanted to add several rows at once, one solution is to [`append!`](@ref) one `DataFrame` to another. For example:
+
+```jldoctest dataframe
+julia> df1 = DataFrame(A=1:3, B=1:3);
+
+julia> df2 = DataFrame(A=4.0:6.0, B=4:6);
```

Comment:
Maybe add a following line with `append!(df1, df2, promote=true)` in `append!` to show that in that case the columns got promoted?